packages/venia-ui/lib/components/CheckoutPage/OrderConfirmationPage/createAccount.js

```diff
@@ -1,21 +1,148 @@
-import React from 'react';
+import React, { useCallback } from 'react';
+import { Form } from 'informed';
+import { useToasts } from '@magento/peregrine';
+import { useCreateAccount } from '@magento/peregrine/lib/talons/CreateAccount/useCreateAccount';
 
-import classes from './orderConfirmationPage.css';
+import combine from '../../../util/combineValidators';
+import { mergeClasses } from '../../../classify';
+import {
+    hasLengthAtLeast,
+    isRequired,
+    validatePassword
+} from '../../../util/formValidators';
+
+import Button from '../../Button';
+import Checkbox from '../../Checkbox';
+import Field from '../../Field';
+import TextInput from '../../TextInput';
+import defaultClasses from './createAccount.css';
+
+import CREATE_ACCOUNT_MUTATION from '../../../queries/createAccount.graphql';
+import CREATE_CART_MUTATION from '../../../queries/createCart.graphql';
+import GET_CART_DETAILS_QUERY from '../../../queries/getCartDetails.graphql';
+import GET_CUSTOMER_QUERY from '../../../queries/getCustomer.graphql';
+import SIGN_IN_MUTATION from '../../../queries/signIn.graphql';
+
+const CreateAccount = props => {
+    const classes = mergeClasses(defaultClasses, props.classes);
+
+    const [, { addToast }] = useToasts();
+
+    const onSubmit = useCallback(() => {
+        // TODO: Redirect to account/order page when implemented.
+        window.scrollTo({
+            left: 0,
+            top: 0,
+            behavior: 'smooth'
+        });
+
+        addToast({
+            type: 'info',
+            message: 'Account successfully created.',
+            timeout: 5000
+        });
+    }, [addToast]);
+
+    const talonProps = useCreateAccount({
```

Comment:
Similarly, I re-used this talon but not the generic `CreateAccount` component to which it belongs. I think that's a no-no from our previous guidance. Different from `ItemsReview`, I actually want a slightly different UI than the old `CreateAccount` view. However the entirety of the logic can be re-used with exception of `isSignedIn` which this new component doesn't care about.